### ISSUE TITLE
where is default

### ISSUE THREAD

**Issue by shuitao** [illacloud/deploy-illa-manually#4]
just deploy an illa, I can see the login page
where can I find the default user account?

**Comment by naj1n** [comment]
@shuitao u should create an account that the mail service used here is provided by illacloud. But don't worry, ur data(username and password etc.) will be saved on your machine.

**Comment by shuitao** [comment]
when register accout,sending code by email, it report ‘网络异常’

**Comment by naj1n** [comment]
We'll fix it ASAP.

**Comment by duanshuaimin** [comment]
> just deploy an illa, I can see the login page where can I find the default user account?

the same problem

**Comment by Allforgot** [comment]
> when register accout,sending code by email, it report ‘网络异常’

same

**Comment by yblis** [comment]
> > when register accout,sending code by email, it report ‘网络异常’
> 
same

**Comment by karminski** [comment]
@shuitao @duanshuaimin @Allforgot @yblis 
Hi, 
We released the version 1.4.0. it include the bug fix.

Please pull the latest docker image and try out.

And if deploy on server/virtual machine, don't forget set the environment variables in ```scripts/run.sh```: 

```bash
# builder-backend api server serve address
API_SERVER_ADDRESS=your_server_ip_address

# builder-backend websocket server serve address
WEBSOCKET_SERVER_ADDRESS=your_server_ip_address
```

For more details, please check out the [README.md](https://github.com/illacloud/deploy-illa-manually/blob/main/docker/README.md).

**Comment by karminski** [comment]
Issue closed due to no feedback over 5 days.
